sapl/relatorios/views.py

```diff
@@ -498,37 +498,36 @@ def remove_html_comments(text):
     return clean_text if len(clean_text) > 0 else text
 
 
+def is_empty(value):
+    if not value:
+        return True
+
+    txt = re.sub(r'\s+|<br.*/>|\n', '', value)
```

Comment:
```suggestion
    txt = re.sub(r'\s+|<br.*/>|\n|&nbsp;', '', value)
```
Esqueci de adicionar o `&nbsp;` ... 🙈